website/docs/r/workflow.html.markdown

```diff
@@ -159,18 +167,32 @@ resource "newrelic_workflow" "workflow-example" {
     name = "Filter-name"
     type = "FILTER"
 
-    predicates {
+    predicate {
+      attribute = "accumulations.policyName"
+      operator = "EXACTLY_MATCHES"
+      values = [ "my_policy" ]
+    }
+
+    predicate {
       attribute = "accumulations.sources"
       operator = "EXACTLY_MATCHES"
       values = [ "newrelic" ]
     }
   }
 
-  destination_configuration {
+  destination {
     channel_id = newrelic_notification_channel.webhook-channel.id
   }
 }
 ```
 
 ## Additional Information
 More details about the workflows can be found [here](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/).
+
+## V1 changes:
+In workflow resource V1 we renamed the following arguments:
```

Comment:
Can you change this to 
`## v3.3 changes`
`In version v3.3 we renamed the following arguments:`